curvefs/src/mds/mds.h

```diff
@@ -25,18 +25,29 @@
 
 #include <memory>
 #include <string>
+
+#include "curvefs/src/mds/chunkid_allocator.h"
 #include "curvefs/src/mds/fs_manager.h"
 #include "src/common/configuration.h"
+#include "src/snapshotcloneserver/snapshotclone_server.h"
 
 using ::curve::common::Configuration;
 
 namespace curvefs {
 namespace mds {
 
+using curve::kvstorage::EtcdClientImp;
+using curve::snapshotcloneserver::InitEtcdConf;
```

Comment:
should be split this InitEtcdConf to a single module?
it is better to add a TODO here now.

Reply:
I think it is a good choice to split  InitEtcdConf to a single module， otherwise the init function will be hard to maintain.